configserver/src/main/java/com/yahoo/vespa/config/server/maintenance/ApplicationPackageMaintainer.java

```diff
@@ -129,4 +131,13 @@ private void createLocalSessionIfMissing(ApplicationId applicationId, long sessi
             sessionRepository.createLocalSessionFromDistributedApplicationPackage(sessionId);
     }
 
+    private void logFailure(Session session, FileReference fileReference, ApplicationId applicationId) {
+        var deployed = Instant.ofEpochSecond(session.getMetaData().getDeployTimestamp());
```

Comment:
You document deployTimestamp as being in milliseconds since Epoch but use ofEpochSecond?

Reply:
Thanks, fixed